src/main/azure-functions-language-worker-protobuf/src/proto/FunctionRpc.proto

```diff
@@ -5,6 +5,7 @@ option java_multiple_files = true;
 option java_package = "com.microsoft.azure.functions.rpc.messages";
 option java_outer_classname = "FunctionProto";
 option csharp_namespace = "Microsoft.Azure.WebJobs.Script.Grpc.Messages";
+option go_package ="github.com/Azure/azure-functions-go-worker/internal/rpc";
```

Comment:
Can you help me understand what this is for?

Reply:
as of now go support is in private preview.